apps/s_time.c

```diff
@@ -432,6 +378,16 @@ static SSL *doConnection(SSL *scon, const char *host, SSL_CTX *ctx)
             SSL_free(serverCon);
         return NULL;
     }
+#if defined(SOL_SOCKET) && defined(SO_LINGER)
+    else {
```

Comment:
You don't really need `else` here. Starting with straight `{` works and I'd say is more readable in this case.